src/riscv/lib/src/machine_state/block_cache/state.rs

```diff
@@ -157,6 +157,14 @@ impl<M: ManagerBase> PartialBlock<M> {
                     return result;
                 }
 
+                Ok(ProgramCounterUpdate::Relative(offset)) => {
```

Comment:
For the `Set` case you may want to copy the comment from the other function here as well